fineract-provider/src/main/java/org/apache/fineract/portfolio/savings/data/SavingsAccountTransactionDataValidator.java

```diff
@@ -179,6 +183,11 @@ public void validateClosing(final JsonCommand command, final SavingsAccount acco
                     account.getId());
         }
 
+        if (account.getSubStatus().equals(400) || account.getSubStatus().equals(500) || account.getSubStatus().equals(600)) {
```

Comment:
I know probably these magic numbers are all over the code but could we start extracting them?
If I just look at the code, 400, 500, 600 doesn't represent anything meaningful for me.